liquidhaskell-boot/src/Language/Haskell/Liquid/Bare/Axiom.hs

```diff
@@ -79,55 +79,71 @@ makeAssumeReflectAxioms src env tycEnv name spSig spec = do
   -- Send an error message if we're redefining a reflection
   case findDuplicatePair val reflActSymbols <|> findDuplicateBetweenLists val refSymbols reflActSymbols of
     Just (x , y) -> Ex.throw $ mkError y $ "Duplicate reflection of " ++ show x ++ " and " ++ show y
-    Nothing -> return $ makeAssumeReflectAxiom spSig env embs name <$> Ms.asmReflectSigs spec
+    Nothing -> return $ turnIntoAxiom <$> Ms.asmReflectSigs spec
   where
+    turnIntoAxiom (actual, pretended) = case makeAssumeReflectAxiom spSig env embs name (Nothing, Nothing) (actual, pretended) of
```

Comment:
No need to pass `(Nothing, Nothing)` as input to `makeAssumeReflectAxiom`. This is the only call site of the function now.